assets/src/components/higher-order/with-amp-story-settings.js

```diff
@@ -253,7 +253,7 @@ export default createHigherOrderComponent(
 										onChange={ setFontSize }
 									/>
 								) }
-								{ isTextBlock && (
+								{ needsTextSettings && (
```

Comment:
There's already the `needsTextSettings` check above, so this one can be safely removed